Dockerfile.pachtf

```diff
@@ -1,4 +1,4 @@
-FROM scratch
+FROM alpine:3.15
```

Comment:
why the image swap?

Reply:
We assume a temp directory exists in the worker, and this was causing the pipeline to break.  I wasn't sure what else we needed so I just switched to a real base image instead.